tap-gui/tap-gui-rbac/setup-tap-gui-rbac-with-gke-auth.md

```diff
@@ -0,0 +1,133 @@
+# Enable Authorization on Remote GKE Clusters in Tanzu Application Platform GUI using Google Auth
+
+This section describes how to leverage Google Auth to add access-controlled visibility for a remote GKE cluster.
+
+Once the Authorization is enabled, you can view your runtime resources on a remote cluster in Tanzu Application Platform GUI. For more detail, please refer to [View Runtime Resources on Remote Clusters in Tanzu Application Platform GUI](./view-resouces-rbac.md).
+
+
+## <a id="googles-oidc-provider"></a> Leveraging Google Auth
+
+When levearging Google's OIDC provider, the process of enabling Authorization requires fewer steps:
+
+1. Add a redirect config on the OIDC side
```

Comment:
I think this text should be "Create OAuth credentials in Google", to match the section bellow